FusionIIIT/templates/globals/usercard.html

```diff
@@ -33,13 +33,13 @@
                     Sem - {{user.extrainfo.student.curr_semester_no}}
                     {% endif %}
                 </span>
-                <span class="right floated meta">
+                <!--<span class="right floated meta">
```

Comment:
Instead of commenting please delete this code

Reply:
Requested changes are done sir.